### ISSUE TITLE
Screen reader should pronounce group name when navigating with Left/Right arrow through Menu bar (Windows/Linux)

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#24151]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

After #20191, it became possible to switch between menu groups with Left/Right arrow (being in submenu navigation).
But on **Windows** and **Linux**, it doesn't read next group name after moving from submenu of previous group (like it does on macOS)

### Supporting files, videos and screenshots

On **Windows**:

https://github.com/user-attachments/assets/6df7c1e1-1a4f-43a6-b3f2-ec9052a45d63

On **Mac**: 

https://github.com/user-attachments/assets/89ec01b9-4140-497b-8400-b93529e6da2b

### What is the latest version of MuseScore Studio where this issue is present?

4.4

### Regression

No.

### Operating system

Windows/Linux

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by DmitryArefiev** [comment]
FYI @shoogle

**Comment by DmitryArefiev** [comment]
Fixed in #24638

There some doubling text with Narrator, but it should be logged/fixed separately
